src/pymorize/cli.py

```diff
@@ -11,6 +11,8 @@
 from rich.traceback import install as rich_traceback_install
 from streamlit.web import cli as stcli
 
+from pymorize.fesom_1p4.nodes_to_levels import convert
+
```

Comment:
It seems that this is not used, can it be cleaned up, or finalized?

Reply:
as above